README.md

```diff
@@ -9,6 +9,9 @@
 | Lucas Huynh | itslupus | huynhl@myumanitoba.ca
 | Humayra Anjum Rafi | Humayra98 | rafiha@myumanitoba.ca
 
+## Team Leader: 
```

Comment:
I think he wants us to write team leader for each sprint

Reply:
Sprint 1- Adam
Sprint 2 - Lucas
Sprint 3 - Humayra
Sprint 4 -Jay

Is this okay?

Reply:
Yeah it looks good!